arwenmandos/execScenario.go

```diff
@@ -17,7 +17,8 @@ func (ae *ArwenTestExecutor) Reset() {
 func (ae *ArwenTestExecutor) ExecuteScenario(scenario *mj.Scenario, fileResolver fr.FileResolver) error {
 	ae.fileResolver = fileResolver
 	ae.checkGas = scenario.CheckGas
-	err := ae.InitVM(scenario.GasSchedule)
+	ae.traceGas = scenario.TraceGas
+	host, err := ae.InitVM(scenario.GasSchedule)
```

Comment:
Don't return the host here, wrap related functionality in a method in the ArwenTestExecutor